docs/en/observability/synthetics-visualize.asciidoc

```diff
@@ -36,6 +36,53 @@ Additional information, including the specific error and a stack trace, is provi
 [role="screenshot"]
 image::images/synthetic-app-error.png[Synthetics app error]
 
+Select a step name to view the waterfall chart for the synthetic monitor.
+
+[role="screenshot"]
+image::images/synthetics-waterfall.png[Synthetics waterfall chart]
+
+The waterfall chart provides a visualization of every request the page made when
+a user executed it. Each line in the chart represents an HTTP network request and
+helps you quickly identify what resources are taking the longest to load and what
+order they are loading.
+
+The colored bars within each line indicate the time spent per resource. Each color
+within a bar represents a different part of that resource's loading process and also
+includes the time spent downloading content for specific MIME types: HTML, JS, CSS,
```

Comment:
This is the first time I'm seeing `MIME` on this page. Should we say `Multipurpose Internet Mail Extensions (MIME)` to explain it? Not sure how common this initialism is.

Reply:
Yeah, we should call it out. Thanks.